sys/include/chunked_ringbuffer.h

```diff
@@ -25,19 +25,32 @@
 
 #include <stdbool.h>
 #include <stdint.h>
+#include <stddef.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
 /**
  * @brief The maximum number of chunks that can be stored in a Chunked Ringbuffer
- *
+ * @deprecated Use @ref CONFIG_CHUNK_NUM_MAX instead. Will be removed after 2022.07
  */
-#ifndef CHUNK_NUM_MAX
+#ifdef DOXYGEN
 #define CHUNK_NUM_MAX   (4)
 #endif
 
+/**
+ * @brief The maximum number of chunks that can be stored in a Chunked Ringbuffer
+ *
+ */
+#ifndef CONFIG_CHUNK_NUM_MAX
+#  ifdef CHUNK_NUM_MAX
```

Comment:
No need for the backcompat, this module was just recently added

Reply:
OK, removed it!